services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> QuerySet:
+    user = User.objects.create_user(
```

Comment:
I think `get_user_model()` should also be used here

Reply:
You should write `get_user_model().objects.create_user(...)` as this is the best practice you can see in the `checklist.md` file. And the answer to this is also in your post)

Reply:
I agree, use `get_user_model()`